tests/contract/test_type_configuration.py

```diff
@@ -73,7 +74,4 @@ def test_get_hook_configuration_with_invalid_json():
 def test_get_hook_configuration_with_not_exist_file():
     with patch("builtins.open", mock_open()) as f:
         f.side_effect = FileNotFoundError()
-        try:
```

Comment:
These tests don't even make sense as written.  `FileNotFoundError` is suppressed